rainier-core/src/main/scala/com/stripe/rainier/compute/Recording.scala

```diff
@@ -0,0 +1,35 @@
+package com.stripe.rainier.compute
+
+import java.io._
+
+trait Recording {
+  def save(path: String): Unit
```

Comment:
might be useful to break this into a `serialize` that produces a stream of bytes and a `save` that writes them to a file (eg, so that we can stash the bytes in thrift or something instead in a hadoop context)

Reply:
👍